### DISCUSSION TITLE
Update deployments to support latest SvelteKit versions

### DISCUSSION THREAD

**Discussion by koodeau** [vercel/community#1040] [Idea]
Vercel is still using `svelte-kit build` by default which is deprecated.

Breaking changes [in this release](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md#100-next359) Vite CLI now replacing `svelte-kit` for build, dev, serve

**Comment by amyegan** [comment]
Hi @koodeau! I'm still able to deploy SvelteKit apps with the latest version using Vercel's SvelteKit preset. Are you running into any specific problems with this?
